mcp-server/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "open-digger-mcp-server",
+  "version": "1.0.0",
+  "description": "MCP server for OpenDigger data with comprehensive analysis tools, trend analysis, repository comparison capabilities and more.",
+  "license": "Apache-2.0",
+  "author": "Frank Zhao",
+  "contributors": [
+    {
+      "name": "Pratyay Banerjee"
+    }
+  ],
+  "homepage": "https://open-digger.cn/",
+  "bugs": "https://github.com/X-lab2017/open-digger-mcp-server/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/X-lab2017/open-digger-mcp-server.git"
+  },
+  "type": "module",
+  "bin": {
+    "open-digger-mcp-server": "dist/index.js"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE"
+  ],
+  "scripts": {
+    "build": "npm run clean && tsc && chmod +x dist/index.js",
+    "dev": "npm run build && node dist/index.js",
+    "start": "node dist/index.js",
+    "watch": "tsc --watch",
+    "clean": "rm -rf dist",
+    "prepare": "npm run build",
+    "sse:test": "SSE_PORT=3001 npm start",
```

Comment:
The command here should be `npm start SSE_PORT=3001`.

Reply:
I tried `npm run sse:test` on powershell, and get an error report. However, `npm start -- SSE_PORT=3001` works fine: 
<img width="1085" height="369" alt="image" src="https://github.com/user-attachments/assets/ec518ebd-7a87-44ee-8ee0-8a07ef67d877" />